### PR TITLE
Add support vpr branch in bitbucket https://github.com/JuanCrg90/vpr/issues/18

### PR DESCRIPTION
**Pull request by n8chz**
In the case of bitbucket, this command opens a branch page, which is not
the same as the branch's code page.

### CODE REVIEW

**Comment by JuanCrg90** [comment]
could you add a test to verify that this works?

you can take as reference this one
https://github.com/JuanCrg90/vpr/pull/27/files

**Comment by n8chz** [comment]
What is to be asserted in this test?

**Comment by JuanCrg90** [comment]
you need to update this test for the bitbucket scenario
https://github.com/JuanCrg90/vpr/blob/master/spec/url_spec.rb#L53-L60


you can take as reference this PR that I added to test the search_url update #20 
https://github.com/JuanCrg90/vpr/pull/27/files

File: lib/vpr/url.rb [hunk 1]
```diff
@@ -52,13 +50,11 @@ def self.commit_url(commit)
...
-          "/commits/all?search="
-        else
-          "/search?q="
-        end
-      "#{GitParser.repo_url}#{search_param}#{commit}"
+      path = {
+        'github.com': "search?q=",
+        'bitbucket.org': "commits/all?search=",
+      }
+      p "#{GitParser.repo_url}/#{path[GitParser.host.to_sym]}#{commit}"
```
Comment (JuanCrg90) [inline lib/vpr/url.rb]: ?????

Comment (n8chz) [reply lib/vpr/url.rb]: oops

**Pull request merged**